test/kennel/unmuted_alerts_test.rb

```diff
@@ -106,6 +106,15 @@ def result
       result.size.must_equal 0
     end
 
+    it "removes monitors that are covered by matching downtimes" do
+      groups = ["pod:pod3", "pod:pod3,project:foo,team:bar"]
+      monitor[:matching_downtimes] = [
+        { end: nil, start: triggered - 10, groups: groups, active: true, scope: ["pod:pod3"] },
+        { end: nil, start: triggered - 10, groups: [], active: true, scope: ["pod:pod7"] }
```

Comment:
scope does not matter ?

Reply:
I don't believe it matters, since the groups end up being expanded to the full group names, but we may need to use it if there's issues with the above...